apps/prairielearn/src/pages/publicAssessmentQuestionsPreview/publicAssessmentQuestionsPreview.html.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from '@prairielearn/html';
+
+import { AssessmentBadge } from '../../components/AssessmentBadge.html.js';
+import { HeadContents } from '../../components/HeadContents.html.js';
+import { Navbar } from '../../components/Navbar.html.js';
+import { TagBadgeList } from '../../components/TagBadge.html.js';
+import { TopicBadge } from '../../components/TopicBadge.html.js';
+import { type AssessmentQuestionRow } from '../../models/questions.js';
+
+export function InstructorAssessmentQuestions({
+  resLocals,
+  questions,
+}: {
+  resLocals: Record<string, any>;
+  questions: AssessmentQuestionRow[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body>
+        ${Navbar({ resLocals })}
+        <main id="content" class="container-fluid">
```

Comment:
Could we make this narrower? I know it's full width on the instructor page, but we also have quite a few more columns there.

```suggestion
        <main id="content" class="container">
```

Reply:
Applied suggestion